### ISSUE TITLE
'NavBar'

### ISSUE THREAD

**Issue by 61ZM07H36R3ML1N** [judygab/web-dev-projects#53]
In NavBar.js export const NavBar throws an error "Identifier 'NavBar' has already been declared. this is breaking the scroll listener
![identifier declared already](https://github.com/user-attachments/assets/a96587dc-471a-4ccd-bb38-a7d72128173d)

**Comment by adab786** [comment]
Fixed #53 @61ZM07H36R3ML1N @yossijaki  @Trkzi-Omar @hacker-kun

**Comment by 61ZM07H36R3ML1N** [comment]
> Fixed #53 @61ZM07H36R3ML1N @yossijaki @Trkzi-Omar @hacker-kun

what was the issue? because the code doesn't look different?

**Comment by Trkzi-Omar** [comment]
> In NavBar.js export const NavBar throws an error "Identifier 'NavBar' has already been declared. this is breaking the scroll listener
> ![identifier declared already](https://github.com/user-attachments/assets/a96587dc-471a-4ccd-bb38-a7d72128173d)
> 

Of course, we can't import "`NavBar`" and export "`NavBar`" in the same file. 


A fix would be to simply change the name of the component and its usages.


For example: `CustomNavBar`

**Comment by 61ZM07H36R3ML1N** [comment]
> > In NavBar.js export const NavBar throws an error "Identifier 'NavBar' has already been declared. this is breaking the scroll listener
> > ![identifier declared already](https://private-user-images.githubusercontent.com/20850665/361562386-a96587dc-471a-4ccd-bb38-a7d72128173d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzNzY0MjYsIm5iZiI6MTcyNTM3NjEyNiwicGF0aCI6Ii8yMDg1MDY2NS8zNjE1NjIzODYtYTk2NTg3ZGMtNDcxYS00Y2NkLWJiMzgtYTdkNzIxMjgxNzNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDE1MDg0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5ZDIwMTdhMmFkYWNlMDA2YTNjNDM3NDdhMmQyN2I0NTM3OTIwMWI1N2UxMTlmZjU5YWRmZmNhOGQwM2Q0NDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MouB1VaWJM4l-PzdmgfoXvMRGNbJB6Qr451ozsLJEQY)
> 
> Of course, we can't import "`NavBar`" and export "`NavBar`" in the same file.
> 
> A fix would be to simply change the name of the component and its usages.
> 
> For example: `CustomNavBar`

Thank you for your response
